packages/babel/src/transformation/file/options/config.json

```diff
@@ -240,6 +236,12 @@
     "description": "set `sources[0]` on returned source map"
   },
 
+  "inputSourceMap": {
+    "type": "boolean",
+    "description": "use existing source map for source map generation - only works with inline source maps",
```

Comment:
@sebmck, I guess I don't fully understand the intent.  For example, I am using TypeScript that generates an inline source map, that I am then running through `babel` on the CLI to translate the ES6 modules.

Is there a different API/mechanism I should be using to achieve this?